esignet-insurance.properties

```diff
@@ -636,25 +636,25 @@ logging.level.io.mosip.esignet=DEBUG
 
 mosip.esignet.authenticator.sunbird-rc.auth-factor.kbi.individual-id-field=policyNumber
 mosip.esignet.authenticator.sunbird-rc.auth-factor.kbi.field-details={{"id":"policyNumber", "type":"text", "format":""},{"id":"fullName", "type":"text", "format":""},{"id":"dob", "type":"date", "format":"dd/mm/yyyy"}}
-mosip.esignet.authenticator.sunbird-rc.auth-factor.kbi.registry-search-url=http://10.3.148.107/registry/api/v1/Insurance/search
+mosip.esignet.authenticator.sunbird-rc.auth-factor.kbi.registry-search-url=http://demo-registry:8081/api/v1/Insurance/search
 mosip.esignet.authenticator.sunbird-rc.kbi.entity-id-field=osid
 
 mosip.esignet.vciplugin.sunbird-rc.enable-psut-based-registry-search=false
-mosip.esignet.vciplugin.sunbird-rc.credential-type.InsuranceCredential.registry-search-url=http://10.3.148.107/registry/api/v1/Insurance/search
-mosip.esignet.vciplugin.sunbird-rc.issue-credential-url=http://10.3.148.107/credential/credentials/issue
+mosip.esignet.vciplugin.sunbird-rc.credential-type.InsuranceCredential.registry-search-url=http://demo-registry:8081/api/v1/Insurance/search
+mosip.esignet.vciplugin.sunbird-rc.issue-credential-url=http://demo-registry-credentials-service:3000/credentials-service/credentials/issue
 mosip.esignet.vciplugin.sunbird-rc.supported-credential-types=LifeInsuranceCredential,InsuranceCredential,VehicleInsuranceCredential
-mosip.esignet.vciplugin.sunbird-rc.credential-type.InsuranceCredential.static-value-map.issuerId=did:web:holashchand.github.io:test_project:32b08ca7-9979-4f42-aacc-1d73f3ac5322
+mosip.esignet.vciplugin.sunbird-rc.credential-type.InsuranceCredential.static-value-map.issuerId=did:web:registry.dev1.mosip.net:identity-service:3e432fe5-bdab-4717-8eaa-a80d79823e58
```

Comment:
Is this resolvable using external did resolver ?

Reply:
the domain is now public and did is resolvable